injectors/linux.js

```diff
@@ -7,33 +7,62 @@ const { BasicMessages, AnsiEscapes } = require('./log');
 // This is to ensure the homedir we get is the actual user's homedir instead of root's homedir
 const homedir = execSync('grep $(logname) /etc/passwd | cut -d ":" -f6').toString().trim();
 
-const KnownLinuxPaths = Object.freeze([
-  '/usr/share/discord-canary',
-  '/usr/lib64/discord-canary',
-  '/opt/discord-canary',
-  '/opt/DiscordCanary',
-  `${homedir}/.local/bin/DiscordCanary` // https://github.com/powercord-org/powercord/pull/370
-]);
+const KnownLinuxPaths = {
+  stable: Object.freeze([
+    '/usr/share/discord',
+    '/usr/lib64/discord',
+    '/opt/discord',
+    '/opt/Discord',
+    `${homedir}/.local/bin/Discord`
+  ]),
+  ptb: Object.freeze([
+    '/usr/share/discord-ptb',
+    '/usr/lib64/discord-ptb',
+    '/opt/discord-ptb',
+    '/opt/DiscordPTB',
+    `${homedir}/.local/bin/DiscordPTB`
+  ]),
+  canary: Object.freeze([
+    '/usr/share/discord-canary',
+    '/usr/lib64/discord-canary',
+    '/opt/discord-canary',
+    '/opt/DiscordCanary',
+    `${homedir}/.local/bin/DiscordCanary` // https://github.com/powercord-org/powercord/pull/370
+  ]),
+  dev: Object.freeze([
+    '/usr/share/discord-development',
+    '/usr/lib64/discord-development',
+    '/opt/discord-development',
+    '/opt/DiscordDevelopment',
+    `${homedir}/.local/bin/DiscordDevelopment`
+  ])
+};
 
+const ProcessRegex = {
+  stable: /discord$/i,
+  ptb: /discord-?ptb$/i,
+  canary: /discord-?canary$/i,
+  dev: /discord-?development$/i
+}
 
-exports.getAppDir = async () => {
+exports.getAppDir = async (platform) => {
   const discordProcess = execSync('ps x')
     .toString()
     .split('\n')
     .map(s => s.split(' ').filter(Boolean))
-    .find(p => p[4] && (/discord-?canary$/i).test(p[4]) && p.includes('--type=renderer'));
+    .find(p => p[4] && (ProcessRegex[platform]).test(p[4]) && p.includes('--type=renderer'));
 
   if (!discordProcess) {
-    let discordPath = KnownLinuxPaths.find(path => existsSync(path));
+    let discordPath = KnownLinuxPaths[platform].find(path => existsSync(path));
     if (!discordPath) {
       const readlineInterface = readline.createInterface({
         input: process.stdin,
         output: process.stdout
       });
 
       const askPath = () => new Promise(resolve => readlineInterface.question('> ', resolve));
-      console.log(`${AnsiEscapes.YELLOW}Failed to locate Discord Canary installation folder.${AnsiEscapes.RESET}`, '\n');
-      console.log('Please provide the path of your Discord Canary installation folder');
+      console.log(`${AnsiEscapes.YELLOW}Failed to locate Discord${platform  == 'canary' ? ' Canary' : ''} installation folder.${AnsiEscapes.RESET}`, '\n');
+      console.log(`Please provide the path of your Discord${platform  == 'canary' ? ' Canary' : ''} installation folder`);
```

Comment:
the platform should always be printed, just print something like `Failed to locate Discord (platform: ${platform})`